src/java/org/apache/cassandra/cql3/QueryOptions.java

```diff
@@ -53,6 +54,8 @@
 
     public static final CBCodec<QueryOptions> codec = new Codec();
 
+    private static final int UNSET_NOWINSEC = Integer.MIN_VALUE;
```

Comment:
I think this can be `Long.MIN_VALUE` for better clarity, since this value is compared with longs, it doesn't seem to leave the class and it doesn't participate in serialization. 

Keeping it as an int could make readers think that choosing `Integer.MIN_VALUE` over `Long.MIN_VALUE` has anything to do with unsigned ints, which I think isn't the case.

Reply:
Right.